src/elements-format.ts

```diff
@@ -31,16 +31,26 @@ export interface Node {
   metadata?: any;
 }
 
-export interface SourceLocation {
-  /** Line number, zero indexed. */
-  line: number;
-  /** Column number, zero indexed. */
-  column: number;
+export interface SourceRange {
   /**
-   * Path to file, relative to the package base. If not present, is the
-   * element's file.
+   * Path to the file containing the definition of the feature,
+   * relative to the parent feature, or the package if the feature has no parent
+   * (e.g. elements).
+   *
+   * If blank, the feature is defined in the same file as its parent.
```

Comment:
I don't quite grok that. Is a "parent" the same as "super"? Wouldn't a parent be an enclosing Feature, like a Element in a Document?